shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommandType.java

```diff
@@ -114,6 +114,9 @@
 
     CONDITIONAL_BLOCK_END(89, WireCommands.ConditionalBlockEnd::readFrom),
 
+    FLUSH_TO_STORAGE(90, WireCommands.FlushToStorage::readFrom),
```

Comment:
I wrote here the following: https://github.com/pravega/pravega/wiki/PDP-49:-Segment-Store-Admin-Gateway#design
> In WireCommands, the negative ids are mostly idle, whereas all the commands use positive command ids. In the case of adding a new admin-specific command, we suggest using the negative ids available from -126 up to -3, as they are free and we can draw a distinction between regular and admin commands based on ids. Note that adding admin commands would not impact the compatibility of existing Pravega clients and servers.

So, if there is no drawback related, please let's stick to this guideline and use negative ids for the new admin-specific wire commands.

Reply:
using -126, and -125 now.